package/main/daml/Daml.Finance.Interface.Types.Common/daml.yaml

```diff
@@ -4,9 +4,11 @@
 sdk-version: 2.6.0-snapshot.20230216.11444.0.59bc473b
 name: daml-finance-interface-types-common
 source: daml
-version: 1.0.0.99.202323.1
+version: 1.0.99.0.202323.1
 dependencies:
   - daml-prim
   - daml-stdlib
+data-dependencies: null
```

Comment:
pls remove this line (gets added by packell for no reason)

Reply:
done